### PR TITLE
fix errors when verify contracts on test net

### PR DESCRIPTION
**Pull request by buikhacnam**
Hi @PatrickAlphaC , I fixed errors of verifying contracts when deploy them to test net. Please check. Thanks!

### CODE REVIEW

File: deploy/02-deploy-time-lock.ts [hunk 1]
```diff
@@ -10,16 +10,17 @@ const deployTimeLock: DeployFunction = async function (hre: HardhatRuntimeEnviro
   const { deployer } = await getNamedAccounts()
   log("----------------------------------------------------")
   log("Deploying TimeLock and waiting for confirmations...")
+  const args = [MIN_DELAY, [], []]
   const timeLock = await deploy("TimeLock", {
     from: deployer,
-    args: [MIN_DELAY, [], []],
+    args,
```
Comment (krakxn) [inline deploy/02-deploy-time-lock.ts:16]:
```suggestion
    args: args,
```

Syntactical error.

File: deploy/02-deploy-time-lock.ts [hunk 2]
```diff
@@ -10,16 +10,17 @@ const deployTimeLock: DeployFunction = async function (hre: HardhatRuntimeEnviro
...
     log: true,
     // we need to wait if on a live network so we can verify properly
     waitConfirmations: networkConfig[network.name].blockConfirmations || 1,
   })
   log(`TimeLock at ${timeLock.address}`)
   if (!developmentChains.includes(network.name) && process.env.ETHERSCAN_API_KEY) {
-    await verify(timeLock.address, [])
+    await verify(timeLock.address, args, "contracts/governance_standard/TimeLock.sol:TimeLock")
```
Comment (krakxn) [inline deploy/02-deploy-time-lock.ts:23]: Is this syntactically correct? As far as I can recall, the `verify()` function only takes 2 parameters.

**Review by krakxn — COMMENTED** [review]
Suggested changes must be looked into, and needs to be verified for syntactical errors.

File: deploy/02-deploy-time-lock.ts (hunk 2 above)
Comment (buikhacnam) [reply deploy/02-deploy-time-lock.ts:23]: This fixes the error: "Error in plugin @nomiclabs/hardhat-etherscan: More than one contract was found to match the deployed bytecode." We need to specifify our TimeLock Contract. Please see it in helper function.

File: deploy/02-deploy-time-lock.ts (hunk 1 above)
Comment (buikhacnam) [reply deploy/02-deploy-time-lock.ts:16]: No this is not syntax error. This is ES^ shorthand> Please refer https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer

Comment (krakxn) [reply deploy/02-deploy-time-lock.ts:16]: That's fair. 

However, why not make it (`args:`) explicit to reduce ambiguity for debuggers (they will have to check what the parameter references to)?

**Pull request closed without merging**